src/components/atoms/TextFeild/TextFeild.tsx

```diff
@@ -0,0 +1,54 @@
+type Props = {
+  label: string;
+  text: string;
+  labelColor?: string;
+  textColor?: string;
+  labelFontFamily?: string;
+  textFontFamily?: string;
+  labelFontWeight?: number;
+  labelFontSize?: string;
+  textFontWeight?: number;
+  textFontSize?: string;
+  className?: string;
+};
+const TextFeild = (props: Props) => {
+  const {
+    label,
+    text,
+    labelColor = "#555454",
+    textColor = "#7E7D7D",
```

Comment:
for labelColor and textColor please use colors: from tailwind.config.js (If needed, add colors there). 
Also add font family in tailwind.config.js and set default font "Inter" in <Text> component.
 {
        primary: "#0DBAD2",
        text_primary: "#7E7D7D",
        text_bold: "#555454",
        tertiary: "#D20D0D"
      }